client/js/socket-events/more.js

```diff
@@ -6,17 +6,24 @@ import socket from "../socket";
 import store from "../store";
 
 socket.on("more", function (data) {
-	const channel = store.getters.findChannel(data.chan);
+	const channel = store.getters.findChannel(data.chan).channel;
```

Comment:
Do not do this, findChannel _can_ return null.

Reply:
Ahh RIP thx, I'll change it

Reply:
@xPaw would you agree to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

```suggestion
	const channel = store.getters.findChannel(data.chan)?.channel;
```

Reply:
If webpack/babel can deal with it, why not.

Reply:
Should I lift up the `ecmaVersion` to `2020` or use a comment to tell eslint to ignore it? ==> https://eslint.org/blog/2020/07/eslint-v7.5.0-released#optional-chaining-support

Reply:
Yeah, let's bump it to 2020